runtime.md

```diff
@@ -1,5 +1,51 @@
 # Runtime and Lifecycle
 
+## State
+
+The runtime state for a container is persisted on disk so that external tools can consume and act on this information.
+The runtime state is stored in a JSON encoded file.
+It is recommended that this file is stored in a temporary filesystem so that it can be removed on a system reboot.
+On Linux based systems the state information is stored in `/run/oci`.
+The directory structure for a container is `<root>/<containerID>/state.json`.
+
+* **id** (string) ID is the container's ID.
```

Comment:
On Wed, Aug 05, 2015 at 09:19:02AM -0700, Mrunal Patel wrote:
> > +* **id** (string) ID is the container's ID.
> 
> How about all lowercase alphabets and digits? 

That sounds good to me, although we might want to allow hyphens for
folks who want to use UUIDs.  And currently runC defaults to the
bundle's directory name, so we'd need to figure out how we wanted to
map other characters into the valid character set.

Do we want to add requirements for the uniqueness domain?  Unique to
the host seems like a minimum, but we can require universal uniqueness
(e.g. “container IDs must be UUIDs [1]”) or leave that up to the
implementation.  I'd rather leave it up to the implementation, since
as long as UUIDs are *valid* container identifiers, runtime callers
that want universal uniqueness are free to use them.

[1]: https://tools.ietf.org/html/rfc4122